fedora/fas.py

```diff
@@ -134,6 +135,15 @@ async def _user_info(self, evt: MessageEvent, username: str | None) -> None:
             f"GPG Key IDs: {' and '.join(k for k in user['gpgkeyids'] or ['None'])}{NL}"
         )
 
+        if groups:
```

Comment:
I think you could unconditionally include the groups in the response and if there are no groups just have "None". That's a personal opinion though and I'm fine either way.